iotdb/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -291,37 +294,37 @@ public void updatePath() {
     preUpdatePath();
 
     // update the paths of subdirectories in the dataDir
-    if (dataDir.length() > 0 && !dataDir.endsWith(File.separator)) {
-      dataDir = dataDir + File.separatorChar;
+    if (getDataDir().length() > 0 && !getDataDir().endsWith(File.separator)) {
```

Comment:
do we need use getDataDir() here?

Reply:
This is automatically refactored. And I think this won't hurt a bit.